frontend/templates/product/sections/CompatibilityNotesSection/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Heading, SimpleGrid } from '@chakra-ui/react';
+import { Wrapper } from '@ifixit/ui';
+import type { Product } from '@pages/api/nextjs/cache/product';
+
+export type CompatibilityNotesSectionProps = {
+   compatibilityNotes?: Product['compatibilityNotes'] | null;
+};
+
+const MAX_DEFAULT_VISIBLE_DEVICES = 10;
+
+export function SplitCompatibilityNotes({
+   compatibilityNotes,
+}: CompatibilityNotesSectionProps) {
+   const devices = compatibilityNotes?.split(/\r?\n/) ?? [];
```

Comment:
I think I commented on another commit, but I think we should map trim and filter blanks here.